packages/xo-server/src/xapi-stats.js

```diff
@@ -244,7 +229,38 @@ const STATS = {
 }
 
 // -------------------------------------------------------------------
-
+// RRD
+// json: {
+//   meta: {
+//     start: Number,
+//     step: Number,
+//     end: Number,
+//     rows: Number,
+//     columns: Number,
+//     legend: String []
+//   },
+//   data: [{
+//      t: Number,
+//      values: Number[],
+//    }]
```

Comment:
Is there always a single item in `data`?

Reply:
It contains `nItems === nColumns`